ddtrace/propagation/http.py

```diff
@@ -94,47 +97,49 @@ def my_controller(url, headers):
         try:
             normalized_headers = {name.lower(): v for name, v in headers.items()}
 
-            trace_id = HTTPPropagator._extract_header_value(
```

Comment:
Why this rename?

Reply:
Technically unnecessary, but I thought it made sense to be consistent with the extracted ids and sampling_priority and dd_origin, before they're parsed into their respective parameters for `Context()`

Reply:
Hmm I'd leave it out for now because it kind of obscures the change to the default value below. I think it is a good refactor but should just be done separately 🙂